src/main/starlark/core/options/convert.bzl

```diff
@@ -10,11 +10,13 @@ def _to_flags(opts, attr_provider):
         list of flags to add to the command line.
     """
     if not attr_provider:
-        return ""
+        return []
 
     flags = []
     for n, o in opts.items():
         value = getattr(attr_provider, n, None)
+        if value == o.args["default"]:
```

Comment:
Is this logic comparing the input value on the user defined `kt_kotlinc_options` target to the default `attr` value specified in the opts value, and skipping if true?